src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Index/IndexSearchRequestBuilder.cs

```diff
@@ -164,7 +165,8 @@ public IndexSearchRequestBuilder ParseFilters(ISearchPhraseParser phraseParser,
         }
 
         public IndexSearchRequestBuilder ParseFacets(ISearchPhraseParser phraseParser,
-            string facetPhrase,
+            string facetPhrase,
+            string languageCode = null,
```

Comment:
Do not change the signature. Add a new method to the builder. `WithCultureName` and get `cultureName` from the builder.